ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -292,9 +292,104 @@ def ocs_install_verification(timeout=600, skip_osd_distribution_check=False):
             constants.METADATA_POOL, constants.DEFAULT_BLOCKPOOL,
             constants.DATA_POOL
         ]
-        crush_rules = [rule for rule in crush_dump['rules'] if rule['rule_name'] in pool_names]
+        crush_rules = [rule for rule in crush_dump['rules']
```

Comment:
fix parenthesis:
crush_rules = [
    ...
]

Reply:
this func is not part of the PR